index.html

```diff
@@ -66,6 +56,7 @@
 </head>
 
 <body>
+	<div id="noconnection" style="display: block"><h1>No Internet Connection</h1><p>Please, check your internet connection and restart Rambox. If you use a Proxy, please go to Preferences to configure it.</p></div>
```

Comment:
Shouldn't this be 'display: none' so that it doesn't show until the `isOnline` check returns false?

Reply:
You're right, I accidentally committed this to test locally. Fixed now.